pkg/blobinfocache/none/none.go

```diff
@@ -1,4 +1,5 @@
-package blobinfocache
+// Package none implements a dummy BlobInfoCache which records no data.
+package none
```

Comment:
`none.NoCache` seems redundant. Could we leave this in the `blobinfocache` package? It also seems convenient to have a dummy implementation in the same package as the interface.

Reply:
> `none.NoCache` seems redundant.

I'm fine renaming `NoCache`, but couldn't think of anything I liked better.

> Could we leave this in the `blobinfocache` package?

No, because that package has `DefaultCache`, which pulls in all the implementations.  So `NoCache` there would require `NoCache` consumers vendor (or have non-vendor access to) the BoltDB libraties.

> It also seems convenient to have a dummy implementation in the same package as the interface.

The *interface* is over in `types` ;).